Jenkinsfile

```diff
@@ -26,8 +26,10 @@ def runStages() {
         }
 
         gitUtils.stageWithContext("Prepare-db", shortenURL = false) {
-            migrateStatus = sh(script: "bundle exec rake db:migrate --trace", returnStatus: true)
-            sh "bundle exec rake db:test:prepare"
+            migrateStatus = sh(script: "bundle exec rake db:test:prepare --trace", returnStatus: true)
+            container("postgresql") {
+              sh("psql compliance < db/cyndi_setup_test.sql")
+            }
```

Comment:
@vkrizan it really wasn't a big issue once we figured out how to tell Jenkins to use a different container :P Thanks to @Victoremepunto for helping figure this out.